pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -46,23 +45,6 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 	// then populate clusterInfo
 	cluster.Info = mon.PopulateExternalClusterInfo(c.context, c.namespacedName.Namespace)
 
-	// If the user to check the ceph health and status is not the admin,
-	// we validate that ExternalCred has been populated correctly,
-	// then we check if the key (whether admin or not) is encoded in base64
-	if !mon.IsExternalHealthCheckUserAdmin(cluster.Info.AdminSecret) {
-		if !cluster.Info.IsInitializedExternalCred(true) {
-			return errors.New("invalid user health checker credentials")
-		}
-		if !cephconfig.IsKeyringBase64Encoded(cluster.Info.ExternalCred.Secret) {
```

Comment:
@leseb Why did we need to check for a base 64 encoding? The keyring is just a raw string from rook's perspective. If it's invalid, the operator will immediately fail to use it anyway.

Reply:
It's an early validation that allows us to fail in a nice way. This makes sure that the key that was injected by the admin has a correct format. Typically, everything will work until we check the state of the cluster in the goroutine, so 1 minute after the cluster is connected. This will give a false impression of success before failing. So I'd suggest putting it back.

Reply:
Sure, I'll put it back for early detection

Reply:
?